ownership-chain/e2e-tests/tests/util.ts

```diff
@@ -50,3 +50,24 @@ export function describeWithExistingNode(title: string, cb: (context: { web3: We
 	});
 }
 
+const MAX_U96 = new BN('79228162514264337593543950336');  // 2^96 - 1
```

Comment:
could be moved to `config.ts` (although it's a not configured value, but follows our practice of defining constants there)